src/Tizen.NUI/src/public/BaseComponents/TextGeometry.cs

```diff
@@ -212,5 +216,137 @@ public static List<Position2D> GetTextPosition(TextLabel textLabel, int start, i
             CheckSWIGPendingException();
             return list;
         }
+
+        /// <summary>
+        /// Get the rendered size of the line based on a requested value, zero-based. <br />
+        /// if a line contains characters with different directions, multiple sizes will be returned for each block of contiguous characters with the same direction. <br />
+        /// </summary>
+        /// <param name="textEditor">The TextEditor control containing the text.</param>
+        /// <param name="number">The number of the line to get the size for, zero-based.</param>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static List<Size2D> GetLineSize(TextEditor textEditor, int number)
```

Comment:
We prefer interface, 
```suggestion
        public static IList<Size2D> GetLineSize(TextEditor textEditor, int number)
```